glowing_waffles/notebooks/acp.py

```diff
@@ -0,0 +1,107 @@
+class ObserveACP:
+    
+    def __init__(self, name, date):
+        
+        from astropy.coordinates import SkyCoord
+        from astropy.time import Time
+        import datetime
+    
+        self.name = name
+        self.filters = []
+        self.binning = 1
+        
+        self.ra = SkyCoord.from_name(name).ra.deg
+        self.dec = SkyCoord.from_name(name).dec.deg
+        
+        #self.date = str(datetime.datetime.today()).split()[0]
+        self.date = date
+        
+        
+    def wait_until(self, time):
+        self.startTime = time
+        
+        
+    def add_filter(self, filt):
+        
+        if filt in self.filters:
+            print("Filter", filt, "is already in list!")
+        
+        elif isinstance(filt, str):
+            # do the string thing
+            self.filters.append(filt)
+        
+        elif isinstance(filt, str) == False:
```

Comment:
Couldn't this just be an `else`? If it needs to be an `elif` then do

```python
elif not isinstance(filt, str):
```

Reply:
I chose elif because I didn't want the the program to go into the statement in some other unforeseen case that I couldn't think of at the moment. I'll change it to

`elif not isinstance(filt, str):`

for the time being